daffodil-cli/src/main/scala/org/apache/daffodil/Main.scala

```diff
@@ -230,11 +234,18 @@ class CLIConf(arguments: Array[String]) extends scallop.ScallopConf(arguments)
   }
 
   implicit def validateConverter = singleArgConverter[ValidationMode.Type]((s: String) => {
```

Comment:
Do you really want to use an implicit def here? I always suggest avoiding them unless you are defining some domain-specific language, a language feature (e.g., we have a typed-equality library that uses them), etc. 

Is this implicit def somehow required by the idiom this code is working in? If so comments to that effect would help.

Reply:
This is how scallop reccommends adding new converters. The idea being that when we add new args we don't have to specify the converter and the implicit stuff finds the right converter.